src/pull-request-handler.ts

```diff
@@ -202,7 +202,7 @@ export function getPullRequestPlan (
     case MergeStateStatus.BLOCKED:
       return {
         code: 'blocked',
-        message: 'The pull request is blocked by a failing or missing check.',
+        message: 'Merging the pull request is blocked by branch protection rules.\n\nPlease make sure probot-auto-merge is part of "Restrict who can push to matching branches" under Branch protection rules.',
```

Comment:
I'd improve the copy:
```suggestion
        message: 'Merging the pull request is blocked by branch protection rules.\n\nPlease make sure probot-auto-merge has permission to push to this target branch under the "Restrict who can push to matching branches" section in the branch protection rules.',
```

We could also replace `"to this target branch"` with `"to '$target_branch'"` (pseudocode for a variable), if it exists (idk if there's such variable in this scope).